### PR TITLE
gh-111201: Allow bracketed paste to work

### PR DESCRIPTION
**Pull request by pablogsal**
* Issue: gh-111201

### CODE REVIEW

File: Lib/_pyrepl/commands.py [hunk 1]
```diff
@@ -462,3 +462,13 @@ class paste_mode(Command):
...
+
+
+class enable_bracketed_paste(Command):
+    def do(self) -> None:
+        self.reader.paste_mode = True
+        
+class disable_bracketed_paste(Command):
+    def do(self) -> None:
+        self.reader.paste_mode = False
+        self.reader.insert("\n")
```
Comment (lysnikolaou) [inline Lib/_pyrepl/commands.py]: Add empty line.

File: Lib/_pyrepl/simple_interact.py [hunk 2]
```diff
@@ -79,6 +79,16 @@ def showtraceback(self):
...
 
 
+def _enable_bracketed_paste() -> None:
+    sys.stdout.write("\x1b[?2004h")
+    sys.stdout.flush()
+
+
+def _disable_bracketed_paste() -> None:
+    sys.stdout.write("\x1b[?2004l")
+    sys.stdout.flush()
```
Comment (lysnikolaou) [inline Lib/_pyrepl/simple_interact.py]: It feels like these belong in console.prepare and console.restore. Happy to merge this as is though, and improve it after the freeze.

**Review by lysnikolaou — COMMENTED** [review]
A couple of nitty comments. Feel free to merge.

File: Lib/_pyrepl/simple_interact.py (hunk 2 above)
Comment (pablogsal) [reply Lib/_pyrepl/simple_interact.py]: Makes sense, let me move this there

File: Lib/_pyrepl/unix_console.py [hunk 3]
```diff
@@ -525,6 +528,14 @@ def clear(self):
         self.__posxy = 0, 0
         self.screen = []
 
+    def __enable_bracketed_paste(self) -> None:
+        os.write(self.output_fd, b"\x1b[?2004h")
+        self.flushoutput()
```
Comment (lysnikolaou) [inline Lib/_pyrepl/unix_console.py]: Not sure but I think that this is not necessary since you're writing with `os.write` directly.

**Review by ambv — APPROVED** [review]

**Review by lysnikolaou — APPROVED** [review]
LGTM apart from one more nitty comment.

**Comment by hauntsaninja** [comment]
Amazing! I love living in this synthwave future

**Pull request merged**